src/main/kotlin/org/stellar/walletsdk/utils/Operation.kt

```diff
@@ -0,0 +1,26 @@
+package org.stellar.walletsdk.utils
```

Comment:
I'm not sure if this is a proper way to structure helper functions like these, but the idea was to use `Opeation` class for any helpers that are related to operations, and `Transaction` for transactions. Please let me know if there is a better way to handle these. :pray: